main.tf

```diff
@@ -1,5 +1,7 @@
 locals {
-  tfe_workspace = {
+  account_variable_set = {
+    name = var.account_variable_set.name != null ? var.account_variable_set.name : var.name
```

Comment:
var.name -> workload-${var.name} or account-${var.name}? To differentiate with e.g. the inception variable sets

Reply:
Done.